requirements/pytorch/check-avail-strategies.py

```diff
@@ -1,2 +0,0 @@
-if __name__ == "__main__":
-    import deepspeed  # noqa: F401
```

Comment:
this looks like it was here on purpose.

Reply:
agree

Reply:
It used to be useful when we had multiple external strategies in the repo. It's not the case anymore.